internal/definitions/definition_handler_identity_claim.go

```diff
@@ -0,0 +1,211 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package definitions
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hyperledger/firefly/internal/log"
+	"github.com/hyperledger/firefly/pkg/database"
+	"github.com/hyperledger/firefly/pkg/fftypes"
+)
+
+func (dh *definitionHandlers) handleIdentityClaimBroadcast(ctx context.Context, state DefinitionBatchState, msg *fftypes.Message, data []*fftypes.Data, verificationID *fftypes.UUID) (DefinitionMessageAction, error) {
+	var claim fftypes.IdentityClaim
+	valid := dh.getSystemBroadcastPayload(ctx, msg, data, &claim)
+	if !valid {
+		return ActionReject, nil
+	}
+
+	return dh.handleIdentityClaim(ctx, state, msg, &claim, verificationID)
+
+}
+
+func (dh *definitionHandlers) verifyClaimSignature(ctx context.Context, msg *fftypes.Message, identity *fftypes.Identity, parent *fftypes.Identity) (valid bool) {
+
+	author := msg.Header.Author
+	if author == "" {
+		return false
+	}
+
+	var expectedSigner *fftypes.Identity
+	switch {
+	case identity.Type == fftypes.IdentityTypeNode:
```

Comment:
Special case I mentioned above where the parent org signs the node claim directly (no separate verification flow)